app/models/concerns/marc_bound_with_note.rb

```diff
@@ -3,12 +3,18 @@
 ###
 module MarcBoundWithNote
   def bound_with?
-    bound_with_note.present?
+    bound_with_note_for_access_panel.present?
```

Comment:
The effect here is to change how we're determining whether something is a bound_with or not. OK.

Reply:
@cbeer Makes sense.